peer/brontide.go

```diff
@@ -743,6 +743,25 @@ func (p *Brontide) Start() error {
 		}
 	}
 
+	// Send this peer its backup data, if we have it.
```

Comment:
Just to help out future developers working in this area, let's add to this comment that this MUST be sent _after_ the init message and _before_ any channel reestablish message